gym/envs/classic_control/pendulum.py

```diff
@@ -6,6 +6,28 @@
 
 
 class PendulumEnv(gym.Env):
+    """
+    Description:
+        The inverted pendulum swingup problem is a classic
+        problem in the control literature. In this version of the
+        problem, the pendulum starts in a random position, and the
+        goal is to swing it up so it stays upright.
+    Source:
+    Observation:
+        Type: Box(3)
+        Num     Observation               Min                     Max
+        0                                 -1                       1
+        1                                 -1                       1
+        2       Speed                     -8                       8
+    Actions:
+        Type: Box(2)
+        Num   Action
+        0     Apply torque -2
+        1     Apply torque +2
+    Reward:
+    Starting State:
+    Episode Termination:
```

Comment:
Reward, Starting State and Episode Termination are also missing, I'm pretty sure all of that can be easily found in the code.